### ISSUE TITLE
Wrong typings configuration in 5.5

### ISSUE THREAD

**Issue by cyrilletuzi** [ReactiveX/rxjs#2857]
**RxJS version:**
5.5.0-beta.0

**Code to reproduce:**
In TypeScript, use any RxJS class (like `Observable`) or new lettable operators, and use the TypeScript quick fix to auto-import the class or method (in VS Code for example).

**Expected behavior:**
Should propose :
- `Import Observable from "rxjs/Observable"`
- `Import map from "rxjs/operators"`

**Actual behavior:**
In 5.5.0-beta.0, it now proposes  : 
- `Import Observable from "rxjs/_typings"`
- `Import map from "rxjs/_typings/operators"`

**Additional information:**
It was OK before 5.5.

**Comment by benlesh** [comment]
Both of those above require brackets:

```
Import { Observable } from "rxjs/Observable"
Import { map } from "rxjs/operators"
```

**Comment by kwonoj** [comment]
So problem in here is about module resolution path. With named import still, module resolution doesn't look up into actual but one depth to our `package.json` for typings placed so TS language server resolves path like below:

![image](https://user-images.githubusercontent.com/1210596/30817902-f4afe24c-a1ce-11e7-8b02-58211ecb8cfc.png)

Means user no longer able to use auto fix or similar to resolve `correct` path to each import. Technically I guess import to `/_typings/*` wouldn't make differences but also it can be confusing.

**Comment by cyrilletuzi** [comment]
@benlesh Yes, the real imports will have the brackets, it's just the TS quick fix message which is patterned that way, not the issue here.

The problem with multiple levels of exports and re-exports was resolved in TS 2.5. There was similar problems with Angular imports, but everything is OK since TS 2.5. It means it's a RxJS problem.

Again, it was working before RxJS 5.5. So it means there was a modification of how the typings are configured, in a non-standard way.

**Comment by jasonaden** [comment]
@cyrilletuzi I think we have a fix for this. Hopefully pushing a `beta.1` that will correct this issue. Coming shortly.

**Comment by benlesh** [comment]
This should be fixed now in 5.5.0-beta.1 and beta.2
